app/controllers/users.js

```diff
@@ -22,5 +24,34 @@ module.exports = {
         logger.error(`DB: ${error.errors[0]}`);
         res.status(500).json({ error: error.errors[0].message });
       });
+  },
+
+  userNewSession(req, res, next) {
+    const errors = validationResult(req);
+    if (!errors.isEmpty()) {
+      logger.error('User input invalid.');
+      logger.error(errors.array());
+      return res.status(422).json({ errors: errors.array() });
+    }
+    const { email, password } = req.body;
+    User.findOne({ where: { email } })
+      .then(user => {
+        if (!user) {
+          logger.error('Email is not registered.');
+          return res.status(401).json({ error: 'User auth failed. Check your email or password.' });
```

Comment:
Let's move this to the error handling in a new PR as we talked.

Reply:
Done in #13